src/services/google-oauth.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@angular/core';
+import { GooglePlus } from '@ionic-native/google-plus';
+import { NativeStorage } from '@ionic-native/native-storage/ngx';
+import { webClientId } from 'src/environments/env';
```

Comment:
If we add to src/environments/env, we should add the procedure of adding webClientId to the readme since this file is secret!

Reply:
Actually just successfully ran it on my iPhone removing the API key.
I think it's easier just removing it, so we don't all have to go get one each.

Like this:
![image](https://user-images.githubusercontent.com/50372986/77237198-7a59d800-6b9c-11ea-8f88-7c629a2df112.png)

Reply:
We need the webclientid for android phones, for iOS it is not necessary

Reply:
Hum that's why it worked for me!! got it ahah